Globals/Globals.h

```diff
@@ -30,13 +30,41 @@
 #include <Time.h>
 #include <OneWire.h>
 #include <SPI.h>
+
 #if defined(__AVR_ATmega2560__) || defined(__AVR_ATmega1280__)
+#if defined RA_STAR
 #include <Ethernet.h>
 #include <EthernetDHCP.h>
+#endif //  RA_STAR
+#if defined RANET && !defined RA_STAR
 #include <SoftwareSerial.h>
-#endif
```

Comment:
RA_STAR doesn't use SoftwareSerial for RA_NET, it has a wiznet slot dedicated for RA_NET